src/components/form/inputs/TextAreaInputNew.tsx

```diff
@@ -100,10 +100,17 @@ export default function TextAreaInputNew({
           }
           if (!textValue) {
             setShowPlaceholder(true);
+          } else {
+            const trimmedAndSingleSpacedValue = textValue
```

Comment:
put this in a helper function, and write a unit test for this

Reply:
Done